### PR TITLE
Removed old spectrogram page

### PR DESCRIPTION
**Pull request by Nepomuceno**
### What does this PR do?

Remove the old spectrogram page and moved to the new one.

### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you successfully ran tests with your changes locally? Including container validation.
* [x] Have you lint your code locally prior to submission?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [x] Have you labelled this PR correctly? E.G. major, milestone, breaking, breaking-change, minor, feature, enhancement, patch, fix, chore, refactor etc

### CODE REVIEW

File: client/src/api/iqdata/__tests__/Queries.test.tsx
Comment (hannahlspencer) [inline client/src/api/iqdata/__tests__/Queries.test.tsx:1]: Is the functionality that was tested in this file covered elsewhere?

Comment (Nepomuceno) [reply client/src/api/iqdata/__tests__/Queries.test.tsx:1]: This is what the new IQ get block does.

File: client/src/pages/docs/plugins.mdx [hunk 1]
```diff
@@ -90,10 +90,10 @@ docker run -dit -p 8000:8000 --name plugins_container plugins_image
 docker ps
 ```
 
-`docker ps` should return the following, if everything worked.  You should then be able to run the client and set your .env file as discussed above.
+`docker ps` should return the following, if everything worked. You should then be able to run the client and set your .env file as discussed above.
 
-CONTAINER ID   IMAGE           COMMAND                  CREATED          STATUS          PORTS                    NAMES
-12ed73ede6a0   plugins_image   "uvicorn --host 0.0.…"   21 seconds ago   Up 19 seconds   0.0.0.0:8000->8000/tcp   plugins_container
+CONTAINER ID IMAGE COMMAND CREATED STATUS PORTS NAMES
```
Comment (hannahlspencer) [inline client/src/pages/docs/plugins.mdx:95]: In MD this displays weirdly, worth doing it as a table or something for clarity?

Comment (Nepomuceno) [reply client/src/pages/docs/plugins.mdx:95]: Makes Sense I just used prettier to format the files and it did this formatting

Comment (Nepomuceno) [reply client/src/pages/docs/plugins.mdx:95]: Actually this should probably be a code block

Comment (Nepomuceno) [reply client/src/pages/docs/plugins.mdx:95]: Fixed

**Review by hannahlspencer — APPROVED** [review]
loves it

**Pull request merged**